src/types/venues.ts

```diff
@@ -167,6 +167,7 @@ export interface BaseVenue {
   showRangers?: boolean;
   chatTitle?: string;
   showReactions?: boolean;
+  showShoutouts?: boolean;
```

Comment:
This will also needed to be added to `Venue_v2_Base` and similar sorts of places. Not sure if that will have wider impacts if done now.. but at the very least we should be adding an `@debt` comment saying it needs to be done in future.